src/m365/spo/commands/folder/folder-add.ts

```diff
@@ -75,6 +110,14 @@ class SpoFolderAddCommand extends SpoCommand {
       responseType: 'json'
     };
 
+    if (args.options.color !== undefined) {
```

Comment:
Right now we're checking 2 times if `color` is specified.
Let's just construct a request options object, and afterward let's check if `color` is specified to set the URL and body. This way we'll have 1 condition that sets all needed values.